StripeConnect/README.md

```diff
@@ -0,0 +1,49 @@
+# <img src="../readme-images/Connect.svg" width="40" /> StripeConnect iOS SDK
```

Comment:
Should StripeConnect be one word or two words here? I see we typically refer to it as "Stripe Connect" below.

Reply:
Yeah, I think it should be two words based on what we do for FC / Identity

Reply:
```suggestion
# <img src="../readme-images/Connect.svg" width="40" /> Stripe Connect iOS SDK
```